### DISCUSSION TITLE
플레이어와 동일한 NPC 캐릭터를 만들 수 있는 방법이 있을까요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#461] [5. Q&A]
플레이어 ID를 토대로 

ZepetoCharacterCreator.CreateByZepetoId(player.zepetoUserId, spawnInfo, (character: ZepetoCharacter) => {  } );

아래와 같이 NPC 캐릭터를 만들었는데, 캐릭터의 모습이 플레이어와 다르게 나옵니다.
혹시 동일한 캐릭터를 복제하는 법이 있을까요?

![image](https://user-images.githubusercontent.com/5895396/157000414-e313fd37-8261-4345-aa03-e0bd7ba226e2.png)

ZepetoCharacter에서 옷 설정이나 외관 설정이 안 보여서 여쭤봅니다:)

**Comment by pumfit** [comment]
CreateByZepetoId의 첫번째 인자인 zepetoId는 zepetoUserId가 아니라 실제 zepetoId이라서 제페토 앱에서 아이디를 확인하시고 넣어보시면 될 것 같습니다!

**Reply by ryusiin** [reply]
감사합니다:>
성공했어요!
![image](https://user-images.githubusercontent.com/5895396/157152524-5aa11612-414a-4cf6-a371-4af92e4aa9b4.png)
